src/main/java/io/cdap/e2e/pages/actions/CdfConnectionActions.java

```diff
@@ -218,4 +218,43 @@ public static void selectPipelineType(String pipelineType) {
       CdfConnectionLocators.locatorOfPipelineTypeButton(pipelineTypeToSelect)));
   }
 
+  /**
+   * Expand the connections for provided connection type
+   *
+   * @param connectionType i.e. BigQuery / GCS / File
+   */
+  public static void expandConnections(String connectionType) {
+    ElementHelper.clickOnElement(CdfConnectionLocators.locateConnectionType(connectionType));
```

Comment:
If the connection list is expanded by default, for eg, in GCS (in case of CDF), we might end up collapsing it by clicking it, can we add a check to ensure we only click if it is not already expanded?

Reply:
Added check.